setup_deps.py

```diff
@@ -292,7 +285,7 @@ def downloadAndExtract(url, destdir, archiveContentPath=None):
             # Taken from https://stackoverflow.com/a/43094365
             def members(ml):
                 subfolder = osp.normpath(archiveContentPath)
-                l = len(subfolder)
+                len(subfolder)
```

Comment:
so `l` has never been used? why do we need the remaining `len(subfolder)` line?

Reply:
Fixed in 08ef8c8074f03a91a1c2e7b13050eaae3d7c9da9

Reply:
👍